src/components/BalanceToken.js

```diff
@@ -39,7 +39,7 @@ const BalanceToken = ({ amount, symbol, color, size, icon }) => {
           icon || tokenIconUrl(tokenAddress, symbol, network && network.type)
         }
       />
-      {amount !== undefined ? splitAmount(amount.toFixed(3)) : ' - '}
+      {amount !== undefined ? splitAmount(amount, decimals) : ' - '}
```

Comment:
should we use null instead of undefined?

Reply:
We could check for both values but i wouldn't drop the `undefined` check